charts/bulk-scan-payment-processor/Chart.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+appVersion: "1.0"
+description: A helm chart for bulk scan pay processor app
+name: bulk-scan-payment-processor
+home: https://github.com/hmcts/bulk-scan-payment-processor
+version: 0.0.3
```

Comment:
aye, perhaps version `0.0.1` as of now. if not `0.0.0` 😂

Reply:
oops that's from spring boot template :P